index.html

```diff
@@ -43,6 +43,36 @@ <h3 class="content"> I can help you build a product , feature or website Look th
             <button class="chevron"><img src="images/Disabled.svg" alt="chevron-down"></button>
         </footer>
     </section>
+    <section class="mobile-popup-section" id="mobile-popup-section">
+        <div class="mobile-popup" id="mobile-popup">
+            <img id="img-pop" src="" alt="">
+            <li class="close-popup"><i id="xbut" class="fa-solid fa-xmark"></i></li>
+            <div class="content-popup-desktop">
+                <div class="content-head-desktop">
+                    <div>
+                        <h3 id="h3-popup"></h3>
+                    </div>
+                </div>
+                <div class="btn-container-desktop">
+                    <a class="button btn-popup-desktop seeproject" id="btn-1">See live <i class="fa-solid fa-arrow-up-right-from-square"></i></a>
+                    <a class="button btn-popup-desktop seeproject" id="btn-2" href="">See source <i class="fa-brands fa-github"></i></a>
+                </div>
+            </div>
+            <div class="details-popup">
+                <ul>
+                    <li><a id="badge1" href="#"></a></li>
+                    <li><a id="badge2" href="#"></a></li>
+                    <li><a id="badge3" href="#"></a></li>
+                    <li><a id="badge4" href="#"></a></li>
+                </ul>
+            </div>
+            <p id="p-pop"></p>
+            <div class="btn-container">
+                <a class="button btn-popup seeproject" id="btn-1">See live     <i class="fa-solid fa-arrow-up-right-from-square"></i></a>
+                <a class="button btn-popup seeproject" id="btn-2" href="">See source     <i class="fa-brands fa-github"></i></a>
+            </div>
+        </div>
+    </section>
     <section class="works" id="wk">
```

Comment:
- [ ] I think it is one of the main requirements is to create the original cards that you already created in HTML you should  create it by using javascript so kindly create it by that would help you learning how to create HTML by js 🔥🚀💯

![image](https://user-images.githubusercontent.com/58553711/188764823-a7ac4e1d-31a3-4a48-bf57-e03997c86971.png)